README.md

```diff
@@ -3,38 +3,55 @@
+framework for creating Beats, and all the officially supported Beats:
 
-Folder  | Description
+Beat  | Description
 --- | ---
-[libbeat](https://github.com/elastic/beats/tree/master/libbeat) | The Go framework for creating new Beats
-[Filebeat](https://github.com/elastic/beats/tree/master/filebeat) | Lightweight log forwarder to Logstash & Elasticsearch
-[Heartbeat](https://github.com/elastic/beats/tree/master/heartbeat) | Testing service availability using PING based on ICMP, TCP or higher level protocols
+[Filebeat](https://github.com/elastic/beats/tree/master/filebeat) | Tails and ships log files
+[Heartbeat](https://github.com/elastic/beats/tree/master/heartbeat) (experimental) | Monitors service availability by sending active pings over ICMP, TCP, or HTTP
 [Metricbeat](https://github.com/elastic/beats/tree/master/metricbeat) | Fetches sets of metrics from the operating system and services
-[Packetbeat](https://github.com/elastic/beats/tree/master/packetbeat) | Tap into your wire data
-[Winlogbeat](https://github.com/elastic/beats/tree/master/winlogbeat) | Sends Windows Event logs
+[Packetbeat](https://github.com/elastic/beats/tree/master/packetbeat) | Monitors the network and applications by sniffing packets
+[Winlogbeat](https://github.com/elastic/beats/tree/master/winlogbeat) | Fetches and ships Windows Event logs
 
 In addition to the above Beats, which are officially supported by
 [Elastic](elastic.co), the
 community has created a set of other Beats that make use of libbeat but live
 outside of this Github repository. We maintain a list of community Beats
 [here](https://www.elastic.co/guide/en/beats/libbeat/master/community-beats.html).
 
-## Documentation and Getting Help
+## Documentation and Getting Started
 
-You can find the documentation on the [elastic.co
-site](https://www.elastic.co/guide/en/beats/libbeat/current/index.html). If you
-need help, you can open a topic on our [discuss
-forums](https://discuss.elastic.co/c/beats).
+You can find the documentation and getting started guides for each of the Beats
+on the [elastic.co site](https://www.elastic.co/guide/):
+
+* [Beats platform](https://www.elastic.co/guide/en/beats/libbeat/current/index.html)
+* [Filebeat](https://www.elastic.co/guide/en/beats/filebeat/current/index.html)
+* [Metricbeat](https://www.elastic.co/guide/en/beats/metricbeat/current/index.html)
+* [Packetbeat](https://www.elastic.co/guide/en/beats/packetbeat/current/index.html)
+* [Winlogbeat](https://www.elastic.co/guide/en/beats/winlogbeat/current/index.html)
+
+
+## Getting Help
+
+If you need help or hit an issue, please start by opening a topic on our [discuss
+forums](https://discuss.elastic.co/c/beats). Please note that we are keeping the
+GitHub tickets reserved for confirmed bugs and enhancement requests.
+
+
+## Downloads
+
+You can download pre-compiled Beats binaries, as well as packages for the
+supported platforms, from [this page](https://www.elastic.co/downloads/beats).
 
 ## Contributing
```

Comment:
I have a couple of comments about the Contributing section (which you haven't changed):

- Change the second sentence to say, "You can help make the Beats better in many ways: report issues, help us reproduce issues, fix bugs, add functionality, or even create your own Beat.
- Make the link to the developer guide point to the master branch instead of current.